admin-dev/themes/new-theme/scss/components/_button.scss

```
@@ -1,6 +1,6 @@
 .btn {
   &.btn-action {
-    color: #576C72;
+    color: black;
```

```suggestion
    color: #000;
```

Respecting bootstrap standards